Cognite.Extensions/CogniteUtils.cs

```diff
@@ -543,56 +543,6 @@ public CogniteUtilsException(string message, Exception innerException) : base(me
         }
     }
 
-    /// <summary>
-    /// Comparer for CogniteSdk Identity objects
-    /// </summary>
-    public class IdentityComparer : IEqualityComparer<Identity>
```

Comment:
Could anything outside the lib using this? Should it be considered breaking to remove?

Reply:
Yes. That said, anything outside the lib should be _strongly_ encouraged not to use it, since it is now native. I'd go so far as to say that removing it is a good thing in that users will be alerted to the fact that they should no longer use it.

Reply:
The advantage of a compiled language is that it won't break their code unless they do something crazy.

Also the only place it's really used (afaik) is in the OPC-UA extractor.

Reply:
Also for a little while longer I think we'll be making breaking changes. Hence the "beta".

Reply:
As long as it's clearly labeled as non-stable yet, I think breaking is fine :) 

We should also get all of those things out of the way before we do a 1.x release